src/pages/rekey-search/rekey-search.effects.ts

```diff
@@ -23,9 +25,8 @@ export class RekeySearchEffects {
         staffNumber: action.staffNumber,
       };
       return this.rekeySearchProvider.getTest(rekeySearchParams).pipe(
-        map((response: any) => {
-          return new rekeySearchActions.SearchBookedTestSuccess(response);
-        }),
+        map(response => this.compressionProvider.extractTestSlotResult(response.toString())),
```

Comment:
do we have coverage for this in `src/pages/rekey-search/__tests__/rekey-search.effects.spec.ts`?

Reply:
Added the coverage